sos/plugins/gfs2.py

```diff
@@ -36,24 +32,6 @@ def setup(self):
         ])
 
         if self.get_option("gfs2lockdump"):
-            if self._mount_debug():
```

Comment:
Wasn't there some rationale to collect `/sys/kernel/debug/gfs2/*` _only_ under some further condition (i.e. `_mount_debug`)?